File: Bridge Java Server/src/main/java/com/middleware/metamenth/enums/MeterAccumulationFrequency.java
package com.middleware.metamenth.enums;

public enum MeterAccumulationFrequency {
    DAILY("Daily"),
    WEEKLY("Weekly"),
    MONTHLY("Monthly"),
    QUARTERLY("Quarterly"),
    YEARLY("Yearly"),
    NONE(null);

    private final String value;

    MeterAccumulationFrequency(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: Bridge Python Client/metamenth/enumerations/meter_accumulation_frequency.py
from enum import Enum


class MeterAccumulationFrequency(Enum):
    """
    Data accumulation frequency of meters

    Author: Peter Yefi
    Email: peteryefi@gmail.com
    """
    DAILY = "Daily"
    WEEKLY = "Weekly"
    MONTHLY = "Monthly"
    QUARTERLY = "Quarterly"
    YEARLY = "Yearly"
    NONE = None

